iconservice/iconscore/icon_score_base.py

```diff
@@ -68,14 +68,16 @@ def __wrapper(calling_obj: "InterfaceScore", *args, **kwargs):
             raise InvalidInstanceException(
                 FORMAT_IS_NOT_DERIVED_OF_OBJECT.format(InterfaceScore.__name__))
 
-        context = calling_obj.context
+        context = ContextContainer._get_context()
         addr_to = calling_obj.addr_to
+        amount = calling_obj.value
+        calling_obj.value = 0   # reset to zero
```

Comment:
I understand the rationale of this line of code. However, I think we must try to eliminate this kind of code that may cause a possible side effect.  Couldn't we put the value along together when we invoke the inter-call method?

Reply:
It's not possible to pass the value as an argument of the inter-call method. This is because argument of the inter-call method can clash with a reserved keyword for the value.

Dev team is looking for a better way to transfer ICX via interface decorator.